### PR TITLE
Create check-services script

### PR DESCRIPTION
**Pull request by rubo77**
* analyzes all services and processes that should run on the gateway
* shows the maintenance status
* shows the gateway mode

### CODE REVIEW

**Comment by rubo77** [comment]
Maybe we should also execute the nrpe checks  locally sommehow?

**Comment by sargon** [comment]
Additional idea: Maybe we should build the test you design here in a similar way to our nrpe/zabbix tests and have a simple handy frontend, a simple bash script, to call them all. That way we are able to build them whole more flexible.

File: manifests/init.pp [hunk 1]
```diff
@@ -119,7 +119,10 @@
       icvpn_as => $mesh_as;
     }
   }
- 
+  
+  ffnord::resources::ffnord::field {
+    "MESH_CODE": value => "${mesh_code}";
```
Comment (sargon) [inline manifests/init.pp]: No good idea, the mesh_code is not unique in an multi community setup. The value would be random.

Comment (rubo77) [inline manifests/init.pp]: So this should be an array? and then we would have to iterate over that array in the check script?

Comment (sargon) [inline manifests/init.pp]: I think all your use cases could be handled without this variable and instead use find/sed.

**Pull request closed without merging**